src/Altinn.Platform/Altinn.Platform.Storage/Storage/Repository/InstanceRepository.cs

```diff
@@ -222,13 +235,20 @@ private IQueryable<Instance> BuildQueryFromParameters(Dictionary<string, StringV
 
                             break;
 
-                        case "labels":
+                        case "process.ended":
+                            queryBuilder = QueryBuilderForEnded(queryBuilder, queryValue);
+                            break;
+
+                        case "appOwner.labels":
                             foreach (string label in queryValue.Split(","))
                             {
-                                queryBuilder = queryBuilder.Where(i => i.Labels.Contains(label));
+                                queryBuilder = queryBuilder.Where(i => i.AppOwner.Labels.Contains(label));
                             }
 
                             break;
+
+                        default:
+                            throw new ArgumentException($"Unknown query parameter: {queryParameter}");
```

Comment:
The usual way of handling unknown parameters is to ignore them. This means it can be a bit harder to identify typing errors, but I still think that is the right approach.

Reply:
The problem with ignoring is that the query per default returns the broadest scope.